### ISSUE TITLE
Typo in Library navigation bar

### ISSUE THREAD

**Issue by teabag12** [migcontrol/django-migcontrol#248]
under 'Library', remove ':' behind the word 'Library' in the navigation bar

**Comment by benjaoming** [comment]
Does it refer to this colon?

![image](https://github.com/migcontrol/django-migcontrol/assets/374612/55a20a77-ed7e-460a-9ce4-0ef3ac31c433)


Strictly speaking, that's not a typo but a structural element to indicate that "Library" is the upper level and the other ones are nested inside that level.

Removing it, IMO, would require an alternative solution to visualize what's going on.

**Comment by teabag12** [comment]
Yes, that's what it's referring to. Aah yes, that makes sense. Maybe we can change "Library:" to "All" or something like that?

**Comment by benjaoming** [comment]
I'm not sure that I understand how "All" communicates that the other items are nested sub-pages. They don't contain things that the library contains.

The issue might well be that "Library" itself is a list of media, and the other 2 items are different lists of data.

Maybe we need to give that list a name? And then have a landing page for the Library that explains what's going on in the 3 sub sections?

```

                            Library

           /                   |                 \
          /                    |                  \
Articles and books    Border Profiteers        Monthly Reviews
```

**Comment by teabag12** [comment]
Sure thing. I'll meet the others next week and come back to you on this :)
